client/src/components/overview/ImgGallery/Large.jsx

```diff
@@ -61,7 +61,7 @@ function Large(props) {
       useCurrentImg(photos[currentImgIndex + 1]);
     }
   };
-  const onClickThumb = (current, i) => {
+  const onClickThu = (current, i) => {
```

Comment:
Not sure about this var name change; I feel like onClickThumb was more descriptive

Reply:
Thanks John! I agree, I changed it for airbnb styling purposes. The name was too long when I was trying to pass it on as props.

Reply:
airbnb said your name was too long? or the line was too long?

Reply:
Oops, yeah, the line was too long.